sdk/cosmos/azure-cosmos/src/main/java/com/azure/cosmos/implementation/throughputControl/controller/group/global/ThroughputGlobalControlClientItem.java

```diff
@@ -0,0 +1,54 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+package com.azure.cosmos.implementation.throughputControl.controller.group.global;
+
+import com.fasterxml.jackson.annotation.JsonProperty;
+
+import java.time.Duration;
+import java.time.ZoneId;
+import java.time.ZonedDateTime;
+
+public class ThroughputGlobalControlClientItem extends ThroughputGlobalControlItem {
+
+    @JsonProperty(value = "initializeTime", required = true)
+    private String initializeTime;
+
+    @JsonProperty(value = "loadFactor", required = true)
+    private double loadFactor;
+
+    /**
+     * Constructor used for Json deserialization
+     */
+    public ThroughputGlobalControlClientItem() {
+
+    }
+
+    public ThroughputGlobalControlClientItem(
+        String id,
+        String partitionKeyValue,
+        double loadFactor,
+        Duration clientItemExpireInterval) {
+        super(id, partitionKeyValue);
+
+        this.loadFactor = loadFactor;
+        this.initializeTime = ZonedDateTime.now(ZoneId.of("UTC")).toString();
```

Comment:
Have you cosnidered using EpochMs instead?

Reply:
yea, but since initializeTime will never change so I thought a more readable format is better?

Reply:
Makes sense